language/vm/vm_runtime/src/lib.rs

```diff
@@ -123,6 +123,8 @@ pub mod txn_executor;
 pub mod execution_stack;
 #[cfg(not(feature = "instruction_synthesis"))]
 mod execution_stack;
+#[macro_use]
+extern crate mirai_annotations;
```

Comment:
Probably a good reason, but Is this extern crate needed?

Reply:
Yes, this crate defines annotations such as precondition!. It has no impact on the runtime of release builds.